### DISCUSSION TITLE
Vim + NVM

### DISCUSSION THREAD

**Discussion by ajdubovoy** [community/community#16857] [Copilot Conversations] [answered]
I use Copilot on Vim and love it. But, I use `nvm` to manage my `node` version since I have to use different versions on different projects. For coc.nvim, I can set:

```
let g:coc_node_path = '~/.nvm/versions/node/v16.13.1/bin/node'
```

Then, it always uses version 16, regardless of what is needed to build the project.

For Copilot, however, no such setting exists. So I'm stuck with Copilot not working if the node version of my project is <16, unless I separately open a Terminal/tmux pane and set the version separately there, which is cumbersome.

It would be so awesome if there were a setting similar to `g:coc_node_path` for Copilot. Thanks!!!

**Comment by tpope** [comment, marked as answer]
```vim
let g:copilot_node_command = '~/.nvm/versions/node/v16.13.1/bin/node'
```

This setting has existed since day one, but was only added to the documentation recently.

**Reply by ajdubovoy** [reply]
Amazing!!! Wow this is the best haha. Thanks
